src/main/java/com/capgemini/brahma/config/JettyServerCustomizer.java

```diff
@@ -0,0 +1,54 @@
+package com.capgemini.brahma.config;
```

Comment:
We don't need this - see camel-boot for the additional dependency - then you can delete.